Samples/WinMLSamplesGallery/WinMLSamplesGallery/Samples/ImageEffects/ImageEffects.xaml.cs

```diff
@@ -0,0 +1,682 @@
+﻿using Microsoft.AI.MachineLearning;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Media.Imaging;
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Graphics.Imaging;
+using Windows.Media;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.UI;
+using WinMLSamplesGallery.Common;
+
+namespace WinMLSamplesGallery.Samples
+{
+
+    public enum Effect
+    {
+        NotSet = 0,
+        Blur3x3,
+        Blur5x5,
+        Blur7x7,
+        ResizeCubicFill,
+        ResizeCubicFit,
+        ResizeLinearFill,
+        ResizeLinearFit,
+        ResizeNearestFill,
+        ResizeNearestFit,
+        PixelSwizzle_123_321,

```

Comment:
does the alpha channel never participate in the swizzles?

Reply:
@smk2007 can better answer this

Reply:
alpha channel is sliced out